src/components/Folder/FolderList/FolderList.js

```diff
@@ -0,0 +1,41 @@
+import FolderAdd from "../FolderAdd/FolderAdd";
+import "./FolderList.css";
+
+const FolderList = ({ folderList, onSelectFolderList, selectId }) => {
+  const handleClickFolderList = (e) => {
+    const folderId = e.target.id === "null" ? null : parseInt(e.target.id);
+    onSelectFolderList({ name: e.target.textContent, id: folderId });
+  };
+
+  return (
+    <div className="folderList">
+      <div className="folderList-content">
+        <span
+          className={`folderList-button ${
+            selectId === null || isNaN(selectId) ? "selected" : ""
+          }`}
+          onClick={handleClickFolderList}
+          id={null}
```

Comment:
P2:
falsy 값인지 확인하는 방식이면 좋겠네요.

```suggestion
          className={`folderList-button ${
            !selectId ? "selected" : ""
          }`}
          onClick={handleClickFolderList}
```